### ISSUE TITLE
Readability of Playground colour scheme

### ISSUE THREAD

**Issue by pyrmont** [swlkr/janetdocs#36]
When I look at a Playground in JanetDocs, I'm finding it a bit difficult to read.

<img width="814" alt="style" src="https://user-images.githubusercontent.com/41152/106070444-cfb12f00-6147-11eb-828f-6bb2c7f95f7e.png">

I'm not sure if it's because I'm using macOS in Dark Mode. Is this how it's intended to look?

/cc @MikeBeller

**Comment by MikeBeller** [comment]
Yeah.  Not sure what to do.  I am terrible at CSS!  I wonder if I need to import two different themes in the ACE editor?  @pyrmont  What is your sense of how the whole site looks in dark mode?  Trying to understand if it's a playground specific problem, or a tweak is needed to the overall CSS of the site.

**Comment by MikeBeller** [comment]
Also I think this relates to #5
